.rustfmt.toml

```diff
@@ -0,0 +1,13 @@
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = true
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
```

Comment:
`false` is the default for this one, which I changed to `true`, based on our conversations in the Clippy PR.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#use_field_init_shorthand